schemas/HPXMLBaseElements.xsd

```diff
@@ -4880,6 +4880,69 @@
 					<xs:attribute name="dataSource" type="DataSource"/>
 				</xs:complexType>
 			</xs:element>
+			<xs:element minOccurs="0" name="MeasureType">
+				<xs:complexType>
+					<xs:choice>
+						<xs:element name="InstallReplace">
+							<xs:complexType>
+								<xs:sequence>
+									<xs:element ref="extension" minOccurs="0"/>
+								</xs:sequence>
+							</xs:complexType>
+						</xs:element>
+						<xs:element name="AtticEncapsulation">
+							<xs:complexType>
+								<xs:sequence>
+									<xs:element minOccurs="0" name="DuctsMovedToConditionedSpace" type="HPXMLBoolean"/>
+									<xs:element minOccurs="0" ref="extension"/>
+								</xs:sequence>
+							</xs:complexType>
+						</xs:element>
+						<xs:element name="FoundationEncapsulation">
+							<xs:complexType>
+								<xs:sequence>
+									<xs:element minOccurs="0" name="DuctsMovedToConditionedSpace" type="HPXMLBoolean"/>
+									<xs:element minOccurs="0" ref="extension"/>
+								</xs:sequence>
+							</xs:complexType>
+						</xs:element>
+						<xs:element name="HVACRedesign">
+							<xs:complexType>
+								<xs:sequence>
+									<xs:element minOccurs="0" name="DistributionTypeChange" type="HPXMLBoolean"/>
+									<xs:element minOccurs="0" name="PartialDisplacement" type="HPXMLBoolean"/>
+									<xs:element minOccurs="0" name="HotWaterIntegrated">
```

Comment:
We should add documentation of these elements. I've proposed documentation below, but I'm not sure it covers all possibilities.

- DistributionTypeChange=TRUE indicates that an HVAC replacement necessitates a change to the distribution type
- PartialDisplacement=TRUE indicates that the measure references and installed component that partially displaces the load met by an existing component that will remain installed, but meet a smaller load
- HotWaterIntegrated=TRUE indicates that the installed system also supplies domestic hot water